README.md

```diff
@@ -20,6 +20,8 @@ For full documentation, please refer to our [docs](https://aws.github.io/amazon-
 
 All releases can be accessed on our releases [page](https://github.com/aws/amazon-genomics-cli/releases).
 
+The latest nightly build can be accessed here: `s3://healthai-public-assets-us-east-1/amazon-genomics-cli/nightly-build/amazon-genomics-cli.zip`
```

Comment:
Do you mind including the steps to install it via the command line here as well?

Reply:
Added the s3 get-object in the installation doc.